workflow.rst

```diff
@@ -16,9 +16,13 @@ to a property of the object to remember the current place.
     The terminology above is commonly used when discussing workflows and
     `Petri nets`_
 
-The Workflow component does also support state machines. A state machine is a subset
-of a workflow and its purpose is to hold a state of your model. Read more about the
-differences and specific features of state machine in :doc:`/workflow/state-machines`.
+The Workflow component helps you handle different kind of workflows in your application.
+
+.. tip::
+
+    Component does also support state machines. A state machine is a subset
+    of a workflow and its purpose is to hold a state of your model. Read more about the
+    differences and specific features of state machine in :doc:`/workflow/state-machines`.
```

Comment:
I think it could be usefull to explain why a developer should/must/can use this component inside its project.

With a sentence like : Use case: if in your project you are using a $status on an object, with notions of progression/validation, this component can be a solid foundation/wrapper.

wdyt?